File: ASSIGNMENT/C_/loop/q8.c
//q8
//Write a program to find out the max from given number 
    
#include<stdio.h>

int main(){

    int num,max=0,i,n;

    printf("enter 4 number:");
    scanf("%d",&num);

    for(i=1;i<=4;i++){
        n=num%10;
        num=num/10;

       

        if(num>max){
            max=n;
        }
    }
    printf("max num:%d\n",max);




    return 0;
}

File: ASSIGNMENT/PYTHON_/Module – 4 (Advance python programming)/q8.py
# Write a python program to find the longest words.  
file_name='q8.txt'

def number(file_name):
    max_word=''
    with open(file_name,'r') as file:
        linse=file.read().split()
        for word in linse:
            if len(word) > len(max_word):
                max_word=word
    return max_word

print(number(file_name))
    